components/Category/Category.js

```diff
@@ -0,0 +1,369 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { fridgeList, shelfList, freezerList } from "../../itemlist";
+import Add from "../../src/Icons/Add.svg";
+import Remove from "../../src/Icons/Remove.svg";
+import Link from "next/link";
+
+function Category() {
+  const [fridgeItemlist, setFridgeItemlist] = useState(fridgeList);
+  const [shelfItemlist, setShelfItemlist] = useState(shelfList);
+  const [freezerItemlist, setFreezerItemlist] = useState(freezerList);
+  return (
+    <>
+      <CategoryStyled>
+        <CategoryNameStyled>Kühlschrank</CategoryNameStyled>
+        <div>
+          <AddProductButton href="/AddEditCard">
```

Comment:
It's common practise to use kebab-case for urls:

```suggestion
          <AddProductButton href="/add-edit-card">
```

this means also renaming the file in pages :)